.github/workflows/release.yml

```
@@ -9,19 +9,90 @@ on:
-      - name: 🖨 Print release info
+      - uses: actions/checkout@v4
+      - name: 🔗 Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - id: build
         env:
-          RELEASE_TAG_NAME: ${{ github.event.release.tag_name }}
-          RELEASE_NAME: ${{ github.event.release.name }}
+          DOCKER_REPO: ${{ vars.DOCKERHUB_REPO.replace('-staging', '-prod') }}
```

actionlint (1.7.7)

parser did not reach end of input after parsing the expression. 5 remaining token(s) in the input: "(", "STRING", ",", "STRING", ")"

(expression)